<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAboutDialog</class>
 <widget class="QDialog" name="qSlicerAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>About 3D Slicer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QTextBrowser" name="SlicerLinksTextBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTextBrowser" name="CreditsTextBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ctkThumbnailLabel" name="SlicerLabel">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="textPosition">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="transformationMode">
      <enum>Qt::SmoothTransformation</enum>
     </property>
     <property name="selectedColor">
      <color alpha="0">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="ButtonBoxWidget" native="true">
     <layout class="QHBoxLayout" name="ButtonBoxlLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="ButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkThumbnailLabel</class>
   <extends>QWidget</extends>
   <header>ctkThumbnailLabel.h</header>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
